osu.Game/Screens/Play/PlayerSettings/PlayerSettingsGroup.cs

```diff
@@ -15,12 +15,28 @@ public PlayerSettingsGroup(string title)
         {
         }
 
+        private bool expandedByDefault = true;
```

Comment:
What on earth is this logic

Reply:
I was aiming to make it so that menus already expanded on load does not collapse on hover lost, is there a property of the toolbox group that i can use to do this instead of writing this hack?

Reply:
This doesn't work great. If you click the button in the toolbox to toggle expanded while hovering, it will cause an incorrect expanded state when leaving hover.

To fix this, I've moved the logic to the base class and handled explicitly as a hovered state, rather than mutating the `Expanded` bindable. This should work much better and is more standard to how we handle this kind of scenario elsewhere.

Note that this does change the editor toolbox behaviour slightly (as it shares the same base class) – contracted panels will now expand on hover. But in testing, this feels better so I'm just going to go with this.

Reply:
Thank you very much - this was one of the approaches I thought of but decided against because of the affect this would do to the editor toolboxes.